common/s3util.cpp

```diff
@@ -273,7 +273,8 @@ shared_ptr<S3Util> S3Util::BuildS3Util(
           read_ratelimit_mb * 1024 * 1024);
   SDKOptions options;
   Aws::InitAPI(options);
```

Comment:
I took a look again, we will do shutdownAPI() at the deconstructor of S3Util. this means when we create a new s3util, the old shutdownAPi will be called. so we still need to call initApi() again